ThingIFSDK/ThingIFSDKTests/PostNewTriggerForScheduleTests.swift

```diff
@@ -0,0 +1,246 @@
+                                                    triggerID: "0267251d9d60-1858-5e11-3dc3-00f3f0b5",
+                                                    enabled: Bool(true),
+                                                    predicate: predicate,
+                                                    commandID:"")
+                                  ),
+                    mockResponse: MockResponse(
+                                    try! NSJSONSerialization.dataWithJSONObject(
+                                        ["triggerID": "0267251d9d60-1858-5e11-3dc3-00f3f0b5"],
+                                        options: .PrettyPrinted),
+                                    urlResponse: NSHTTPURLResponse(
+                                                   URL: NSURL(string:setting.app.baseURL)!,
+                                                   statusCode: 201,
+                                                   HTTPVersion: nil,
+                                                   headerFields: nil)!,
+                                    error: nil)
+                               )
+            let tag = "testPostNewTrigger_success_\(index)"
+            testCase.verifier.requestVerifier.tag = tag
+            testCase.verifier.callbackVerifier.tag = tag
+            retval.append(testCase)
+        }
+        return retval;
+    }
+
+    func postNewTrigger(testCase: TestCase, setting: TestSetting) {
+        weak var expectation : XCTestExpectation!
+        defer {
+            expectation = nil
+        }
+        expectation = self.expectationWithDescription(
+                          testCase.verifier.requestVerifier.tag!)
+
+        MockSession.requestVerifier =
+            testCase.verifier.requestVerifier.verifier
+        MockSession.mockResponse = testCase.mockResponse
+        iotSession = MockSession.self
+
+        setting.api._target = setting.target
+
+        setting.api.postNewTrigger(
+            testCase.parameter.schemaName,
+            schemaVersion: testCase.parameter.schemaVersion,
+            actions: testCase.parameter.actions,
+            predicate: testCase.parameter.predicate,
+            completionHandler: { (trigger, error) -> Void in
+                testCase.verifier.callbackVerifier.verifier(trigger, error)
+                expectation.fulfill()
+            })
+        self.waitForExpectationsWithTimeout(TEST_TIMEOUT) { (error) -> Void in
+            if error != nil {
+                XCTFail("execution timeout for \(testCase.verifier.requestVerifier.tag)")
+            }
+        }
+    }
+
+    func testPostNewTrigger_success() {
+        let setting = TestSetting()
+
+        let predicates: [Predicate] = [
+            SchedulePredicate(schedule: "1 * * * *"),
+            SchedulePredicate(schedule: "1 1 * * *"),
+            SchedulePredicate(schedule: "1 1 1 * *"),
+            SchedulePredicate(schedule: "1 1 1 1 *"),
+            SchedulePredicate(schedule: "1 1 1 1 1")
+        ]
```

Comment:
I agree. Please add tests for invalid format of `schedule` string, which are not `201` response.